_studio/mfx_lib/encode_hw/mpeg2/src/mfx_mpeg2_encode_utils_hw.cpp

```diff
@@ -822,7 +822,8 @@ namespace MPEG2EncoderHW
                 out->mfx.CodecLevel = MFX_LEVEL_UNKNOWN;
                 bWarning = true;
             }
-            if (CorrectProfileLevelMpeg2(out->mfx.CodecProfile, out->mfx.CodecLevel,
+            if (out->mfx.FrameInfo.FrameRateExtD != 0 &&
```

Comment:
why you put this condition here? You don't do any divisions on this level. I guess you need to put it inside ```CalculateUMCFramerate```

Reply:
I hurried, checking is already done inside the CalculateUMCFramerate function, this PR is not needed